content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -151,7 +152,7 @@ Below are the procedures to set up autocompletion for PowerShell.
 1. Download the latest release with the command:
 
    ```powershell
-   curl -LO "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl-convert.exe"
+   Invoke-WebRequest -Uri "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl-convert.exe" -OutFile "kubectl-convert.exe"
```

Comment:
```suggestion
   curl.exe -LO "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl-convert.exe"
```

Reply:
I think Invoke-Webrequest would be more conventient for native windows users since powershell is the goto command line for most power  users.

Reply:
I'd actually prefer to keep this as `curl.exe`.
`curl` is an alias for `Invoke-WebRequest` in powershell and the arguments are not the same as for curl.exe and we have seen this cause a lot of confusion in the past.